src/InlineRelationsTransformer.cpp

```diff
@@ -612,11 +613,11 @@ NullableVector<AstArgument*> getInlinedArgument(AstProgram& program, const AstAr
         }
     } else if (dynamic_cast<const AstFunctor*>(arg) != nullptr) {
         if (const auto* functor = dynamic_cast<const AstIntrinsicFunctor*>(arg)) {
-            for (size_t i = 0; i < functor->getArity(); i++) {
```

Comment:
can you change this loop to size_t i=0; for(auto arg: fun.getArguments()) { ....; i++}